lib/apis/fetch.js

```diff
@@ -11,7 +11,15 @@ export default (url, options = {}) => {
 
     request(url, opts, (err, response) => {
       if (!!err || response.statusCode !== 200) {
-        return reject(err || new Error(response.body));
+        if (err) return reject(err);
```

Comment:
What's the case where this will not just return a rejection? When would `err` be null?